echopype/tests/echodata/test_zarr_combine.py

```diff
@@ -237,7 +239,7 @@ def test_append_ds_list_to_zarr(append_ds_list_params):
 
     # create temporary directory for zarr store
     temp_zarr_dir = tempfile.TemporaryDirectory()
-    zarr_path = temp_zarr_dir.name + "/" + zarr_name
+    zarr_path = join_paths(temp_zarr_dir.name, zarr_name)
```

Comment:
For all uses of `join_paths` in this test function `test_append_ds_list_to_zarr`:
- the original form using pure string operation is not a good practice -- should use os.path
- since this test is to test the functionalities of combining datasets into zarr, using `join_paths` here introduces an unintended "test" on the path handling functionality of that function, making the tests hard to follow if anything breaks

I suggest removing the use of `join_paths` here and use os.path for creating the paths you need in this test.

Reply:
One thing I am not sure is if os.path gives you `\` instead of `/` on Windows, is this why you used pure string operations? It seems that using PureWindowsPath you can "simulate" how the string should be interpreted in windows. How about using that?
Below from the 3rd block under this [Operators](https://docs.python.org/3/library/pathlib.html#operators) section:
```python
>>> p = PureWindowsPath('c:/Program Files')
>>> str(p)
'c:\\Program Files'
```

Reply:
As you have suggested, I have replaced the usage of `join_paths` with `os.path.join` in all tests where it is not necessary.